cmd/guacone/cmd/vulnerability.go

```diff
@@ -329,21 +335,23 @@ func searchDependencyPackages(ctx context.Context, gqlclient graphql.Client, top
 	return path, tableRows, nil
 }
 
-func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilitiesResponses []model.VulnerabilitiesVulnerabilitiesVulnerability, edgeType model.Edge, depth, pathsToReturn int) ([]string, error) {
+func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilitiesResponses []model.VulnerabilitiesVulnerabilitiesVulnerability, depth int, pathsToReturn int) ([]string, []table.Row, error) {
 	type vulnNeighbor struct {
 		node model.NeighborsNeighborsNode
 		id   string
 	}
 
 	var path []string
 	var vulnNodeNeighborResponses []vulnNeighbor
+	var tableRows []table.Row
 
+	edgeTypes := []model.Edge{model.EdgeVulnerabilityCertifyVuln, model.EdgeVulnerabilityCertifyVexStatement}
 	for _, vulnerabilitiesResponse := range vulnerabilitiesResponses {
 		for _, vulnerabilityNodeID := range vulnerabilitiesResponse.VulnerabilityIDs {
-			vulnNodeNeighborResponse, err := model.Neighbors(ctx, gqlclient, vulnerabilityNodeID.Id, []model.Edge{edgeType})
+			vulnNodeNeighborResponse, err := model.Neighbors(ctx, gqlclient, vulnerabilityNodeID.Id, edgeTypes)
 
 			if err != nil {
-				return nil, fmt.Errorf("error querying neighbor for vulnerability: %w", err)
+				return nil, []table.Row{}, fmt.Errorf("error querying neighbor for vulnerability: %w", err)
```

Comment:
replace `[]table.Row{}` with `nil` in all the returns. A nil slice is valid, no need to allocated a 0 length slice.